backend/workspaces/views.py

```diff
@@ -392,131 +403,139 @@ def user_analytics(self, request, pk=None):
             if tgt_language != None and tgt_language not in list_of_user_languages:
                 continue
 
-            if email == ws_owner or email == org_owner :
+            if email == ws_owner or email == org_owner:
                 continue
 
-            if tgt_language == None :
-                projects_objs = Project.objects.filter(workspace_id=pk, users = each_user,project_type = project_type)
-            else :
+            if tgt_language == None:
+                projects_objs = Project.objects.filter(workspace_id=pk, users=each_user, project_type=project_type)
+            else:
                 selected_language = tgt_language
-                projects_objs = Project.objects.filter(workspace_id=pk, users = each_user,project_type = project_type,tgt_language = tgt_language)
+                projects_objs = Project.objects.filter(
+                    workspace_id=pk, users=each_user, project_type=project_type, tgt_language=tgt_language
+                )
             project_count = projects_objs.count()
-            proj_ids = [eachid['id'] for eachid in projects_objs.values('id')]
-            
-    
-            all_tasks_in_project = Task.objects.filter(Q(project_id__in=proj_ids) & Q(annotation_users= each_user ))
+            proj_ids = [eachid["id"] for eachid in projects_objs.values("id")]
+
+            all_tasks_in_project = Task.objects.filter(Q(project_id__in=proj_ids) & Q(annotation_users=each_user))
             assigned_tasks = all_tasks_in_project.count()
-            
-
-            annotated_tasks_objs =Task.objects.filter(Q(project_id__in=proj_ids) & Q(annotation_users= each_user )\
-                    & Q(task_status__in = ['accepted','rejected','accepted_with_changes','labeled']))
-
-            annotated_task_ids = list(annotated_tasks_objs.values_list('id',flat = True))
-            annotated_labeled_tasks =Annotation.objects.filter(task_id__in = annotated_task_ids ,parent_annotation_id = None,\
-                created_at__range = [start_date, end_date])
-                
-            annotated_tasks = annotated_labeled_tasks.count()
-                
-            lead_time_annotated_tasks = [ eachtask.lead_time for eachtask in annotated_labeled_tasks]
+
+            annotated_tasks_objs = Task.objects.filter(
+                Q(project_id__in=proj_ids)
+                & Q(annotation_users=each_user)
+                & Q(task_status="accepted")
```

Comment:
It should be filtered considering all task statuses. ['accepted','rejected','accepted_with_changes','labeled']))